modules/extensions-spring-support/src/main/resources/META-INF/mule-extension.xsd

```diff
@@ -0,0 +1,113 @@
+        <xsd:annotation>
+            <xsd:documentation>
+                Application level configuration about how to handle Mule Extensions
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
+    <xsd:complexType name="extensionsConfigType">
+        <xsd:complexContent>
+            <xsd:extension base="mule:abstractConfigurationExtensionType">
+                <xsd:sequence>
+                    <xsd:choice>
+                        <xsd:element name="dynamic-configuration-expiration" type="dynamicConfigurationExpirationType" minOccurs="0" maxOccurs="1">
+                            <xsd:annotation>
+                                <xsd:documentation>
+                                    Configuration about how often to check for expirable dynamic configuration instances and actually do
+                                    dispose them
+                                </xsd:documentation>
+                            </xsd:annotation>
+                        </xsd:element>
+                    </xsd:choice>
+                </xsd:sequence>
+            </xsd:extension>
+        </xsd:complexContent>
+    </xsd:complexType>
+
+    <xsd:complexType name="dynamicConfigurationExpirationType">
+        <xsd:attribute name="frequency" type="mule:substitutableLong" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    A scalar time value for how often should the platform check for expirable dynamic configs
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="timeUnit" type="mule:timeUnitType" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    A time unit that qualifies the frequency attribute
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+    </xsd:complexType>
+
+    <xsd:element name="dynamic-config-policy" type="dynamicConfigPolicyType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Settings which define how the platform should manage dynamic instances of a configuration element.
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
+    <xsd:complexType name="dynamicConfigPolicyType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Settings which define how the platform should manage dynamic instances of a configuration element.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:sequence>
+            <xsd:choice>
+                <xsd:element name="expiration-policy" type="expirationPolicyType" minOccurs="0" maxOccurs="1">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                            Configures the maximum amount of time that a dynamic configuration instance can remain idle
```

Comment:
nop.. It's up to how much time will you allow config instances to slack before being purge. It's a maximum not a minimum. The minimum will always be zero